src/logiDice.js

```diff
@@ -73,13 +73,16 @@ module.exports = {
 			return Promise.all(recPromises).then((subQueries) => {
 				let diceResult = subQueries.reduce((tally, current, index) => {
 					return mergeResults(tally, current);
-				}, false);
+				}, {
+					rolls: [],
```

Comment:
oh....... the `x` case fails completely because it does a further regexp that only accepts `\d+d\d+` which the degenerate case does not include.

guard should be updated....

will work on that.